src/constants/MarkdownFiles/posts/2025-06-08-gsoc-25-diwangshu-week01.md

```diff
@@ -4,8 +4,7 @@ excerpt: "Deploying a Streamlit app for testing"
 category: "DEVELOPER NEWS"
 date: "2025-06-04"
 slug: "2025-06-04-gsoc-25-Diwangshu-week01"
-author: "Diwangshu Kakoty"
-description: "GSoC'25 Contributor working on AI Tool for Reflection"
+author: "@/constants/MarkdownFiles/authors/diwangshu-kakoty.md"
```

Comment:
This doesn't seem to be rendering properly.

![Screenshot from 2025-06-15 09-01-53](https://github.com/user-attachments/assets/5b74226b-d7f9-4e80-bdf3-dcf31b5b6e53)


Please review documentation, make changes, and test them.